2DPlatformerController/Assets/ScoreScript.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+public class ScoreScript : MonoBehaviour
+{
+    public static int scoreValue;
+    Text score;
+    // Start is called before the first frame update
+    void Start()
```

Comment:
Add access modifier. If one isn't listed, it will default to private, but it's better to be explicit about it